sirepo/uri.py

```diff
@@ -74,21 +74,15 @@ def local_route(sim_type, route_name=None, params=None, query=None):
             if not params or p not in params:
                 continue
         u += '/' + _to_uri(params[p])
-    return app_root(t) + '#' + u + _query(query)
-
-
-def server_root(external):
-    return flask.url_for('_dispatch_empty', _external=external)
+    return app_root(t, external=external) + '#' + u + _query(query)
 
 
 def server_route(route_or_uri, params, query):
     """Convert name to uri found in SCHEMA_COMMON
-
```

Comment:
Leave blank line between description and Args.

Reply:
ad61fc813610afe4d4c57cbe0e4322c503d6ac6c